include/openthread/thread.h

```diff
@@ -138,6 +138,11 @@ typedef struct
     uint8_t      mAge;                 ///< Time last heard
     bool         mAllocated : 1;       ///< Router ID allocated or not
     bool         mLinkEstablished : 1; ///< Link established with Router ID or not
+#if OPENTHREAD_CONFIG_MAC_CSL_RECEIVER_ENABLE
+    uint8_t mVersion;          ///< Thread version
+    uint8_t mCslClockAccuracy; ///< CSL clock accuracy
```

Comment:
Doc the unit? I tihnk it is in ppm, right?
```suggestion
    uint8_t mCslClockAccuracy; ///< CSL clock accuracy in ppm
```